src/stories/Library/recommended-material/recommended-material.scss

```diff
@@ -0,0 +1,47 @@
+$_max-height-recommended-material: 432px;
+$_aspect-ratio-mobile: 285 / 320;
+$_aspect-ratio-desktop: 378 / 432;
+$_material-image-box-shadow: 0 4px 40px rgba(0, 0, 0, 0.45);
+
+.recommended-material {
+  background-color: $color__global-secondary;
+
+  max-height: 432px;
+  width: 100%;
+  padding: $s-md;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 1fr auto;
+  row-gap: $s-xl;
+  aspect-ratio: $_aspect-ratio-mobile;
+  grid-area: material;
+
+  @include media-query__medium {
+    padding: $s-lg;
+    aspect-ratio: $_aspect-ratio-desktop;
+    max-width: 378px;
```

Comment:
Please consider if this size should be handled differently.

As far as I can tell it is replicated across the recommendation and the recommended material. Can we use a shared variable or at least a comment?

Reply:
If this is in regards to the aspect ratio, i don't think it is used in both the css files. 

The max-width's will be refactored to variables.